### DISCUSSION TITLE
Change player nickname while player is online (on server)

### DISCUSSION THREAD

**Discussion by sunnamed434** [SmartlyDressedGames/Unturned-3.x-Community#4303] [Ideas]
I heard ideas like sending packet that the player is connected again etc, but I think its not a solution

**Comment by rube200** [comment]
Already asked like 300x but okay

**Comment by DanielWillett** [comment]
I think a lot of people would like to see this feature.

Was discussed here as well:
https://github.com/SmartlyDressedGames/Unturned-3.x-Community/issues/2902
